src/CampaignsAdmin/report/locationCard.css

```diff
@@ -0,0 +1,34 @@
+.location-card {
+  flex: 0 1 auto;
+  /* width: 400px; */
+  background-color: #eff2f5;
```

Comment:
Start abstracting away the colors you're using into variables, there should be support for them on the project: https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties